### DISCUSSION TITLE
希望pcl可以出一个mod文件查重，有时候mod装太多了全是英文也分不清。。。

### DISCUSSION THREAD

**Discussion by lbwnb080114** [Meloong-Git/PCL#1681] [讨论]
希望pcl可以出一个mod文件查重，有时候mod装太多了全是英文也分不清。。。

**Comment by Chiloven945** [comment]
应该可以通过识别modID做到？

**Comment by iPodtouch6557** [comment]
可以交一个issue哦

**Comment by Pigeon0v0** [comment]
游戏启动失败会有提示的

**Comment by WForst-Breeze** [comment]
有重复 mod 按理讲是不会启动成功的，游戏会直接扔给你崩溃报告的

**Reply by Chiloven945** [reply]
有些时候启动成功了，还没有任何问题，我就遇到过(

**Reply by Super-XiaoNian** [reply]
> 有些时候启动成功了，还没有任何问题，我就遇到过(

那不可能的，要么是你装错版本了(指Forge/Fabric)。
